guides/micronaut-mqtt/cli/kotlin/src/test/kotlin/example/micronaut/MicronautguideCommandTest.kt

```diff
@@ -0,0 +1,50 @@
+package example.micronaut
+
+import io.micronaut.configuration.picocli.PicocliRunner
+import io.micronaut.context.ApplicationContext
+import io.micronaut.context.annotation.Replaces
+import io.micronaut.context.env.Environment
+import jakarta.inject.Singleton
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import java.io.ByteArrayOutputStream
+import java.io.OutputStream
+import java.io.PrintStream
+import java.math.BigDecimal
+import java.nio.charset.StandardCharsets.UTF_8
+
+class MicronautguideCommandTest {
+
+    @Test
+    fun testWithCommandLineOption() {
+        val baos: OutputStream = ByteArrayOutputStream()
+        System.setOut(PrintStream(baos))
+
+        ApplicationContext.run(
+                mapOf("spec.name" to "MicronautguideCommandTest"),
```

Comment:
It needs to add `mqtt.enabled", false`. See https://github.com/micronaut-projects/micronaut-guides/pull/817/commits/1d422dc0aaa1a4df7e2cf07df5fa7ea7a249bd40